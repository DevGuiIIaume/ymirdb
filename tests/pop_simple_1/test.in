set keyA 11 22 33 44
get keyA
pop keyA
get keyA
pop keyA
get keyA
pop keyA
get keyA
pop keyA
get keyA
pop keyA
get keyA
bye
