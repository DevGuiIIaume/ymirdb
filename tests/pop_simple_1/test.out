> ok

> [11 22 33 44]

> 11

> [22 33 44]

> 22

> [33 44]

> 33

> [44]

> 44

> []

> nil

> []

> bye
